tests/test_subproblem.py

```diff
@@ -0,0 +1,63 @@
+import cvxpy as cp
+import numpy as np
+import pytest
+
+from ncopt.sqpgs.main import SubproblemSQPGS
+
+
```

Comment:
Are the tested subproblems somehow special (that is, coming from a special case of the underlying problem)?

Reply:
They are the first iterates of the rosenbrock example, so not special.